tests/side_panels/building_blocks/chart_title.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, xml } from "@odoo/owl";
```

Comment:
the file should probably be moved inside the `chart` folder now that it clearly is only related to charts :)